### ISSUE TITLE
Add filtering to import existing movie process

### ISSUE THREAD

**Issue by c0rnflake** [Radarr/Radarr#211]
At the moment the import existing movie process lists all movies including those that already exist.

This is painful when you've a significant library to a) wait on these all parsing and b) to actually scroll through the list to find what you need to work on importing.

Adding a filter to remove already existing movies would be helpful in this scenario

**Comment by wcomartin** [comment]
this could be a duplicate of https://github.com/Radarr/Radarr/issues/150 ?

**Comment by c0rnflake** [comment]
#150 is more geared towards an Add All button, I think these two options combined would be a nice solution to both issues, but I think they're separate issues.  This option would definitely be helpful if an Add All button was added also.

**Comment by onedr0p** [comment]
I bought it up in #150 here https://github.com/Radarr/Radarr/issues/150#issuecomment-272083561

Closing
